src/lib.rs

```diff
@@ -1619,6 +1619,61 @@ impl<B: BitBlock> BitVec<B> {
     pub fn shrink_to_fit(&mut self) {
         self.storage.shrink_to_fit();
     }
+
+    /// Inserts a given bit at index `at`, shifting all bits after by one
+    ///
+    /// # Panics
+    /// Panics if `at > nbits`
+    ///
+    /// # Examples
+    ///```
+    /// use bit_vec::BitVec;
+    ///
+    /// let mut b = BitVec::new();
+    ///
+    /// b.push(true);
+    /// b.push(true);
+    /// b.insert(1, false);
+    ///
+    /// assert_eq!(b.len(), 3);
+    /// assert!(b.eq_vec(&[true, false, true]));
```

Comment:
No need to test for len if you test for eq_vec.

Reply:
Yeah, I just wanted it to be more explicit